development/cpp/variant_class.rst

```diff
@@ -18,10 +18,10 @@ A Variant can:
 -  Store almost any datatype
 -  Perform operations between many variants (GDScript uses Variant as
    its atomic/native datatype).
--  Be hashed, so it can be compared quickly to over variants
+-  Be hashed, so it can be compared quickly to other variants
 -  Be used to convert safely between datatypes
 -  Be used to abstract calling methods and their arguments (Godot
-   exports all its functions through variants)
```

Comment:
This is the correct form

Reply:
The formatting also looks a little off there :) And the commits should probably be squashed.